public/css/icinga/dashboards.less

```diff
@@ -0,0 +1,333 @@
+  margin-top: .6em;
+  border-top: 1px solid @gray-light;
+}
+
+// Add new home button of the dashboard manager
+.controls .add-home {
+  .button();
+  float: right;
+  font-size: 1.2em;
+  font-weight: 500;
+  color: @text-color-on-icinga-blue;
+  background-color: @icinga-blue;
+}
+
+.collapsed .action-link.add-dashlet, .collapsed .action-link.add-dashboard {
+  display: none;
+}
+
+.action-link.add-dashboard {
+  width: 85%;
+  margin: .5em 0 0;
+}
+
+.action-link.add-dashlet, .action-link.add-dashboard {
+  .rounded-corners();
+  order: 1;
+  font-size: 1.2em;
+  font-weight: 500;
+  padding: .6em 1em;
+  text-decoration: none;
+  background-color: @low-sat-blue;
+}
+
+.dashboard-list-control .dashlet-item-list {
+  margin-left: 2.2em;
+  display: flex;
+  width: 97.8%; // 100% - 2.2em margin-left
+
+  .dashlet-list-item {
+    width: 49.7%;
+    margin-bottom: .5em;
+    background: transparent;
+  }
+
+  .dashlet-list-item:nth-child(odd) {
+    margin-right: .5em;
+  }
+}
+
+.dashboard-list-control {
+  width: 85%;
+
+  h1.collapsible-header {
+    margin-left: 1.5em;
+  }
+
+  .action-link.add-dashlet {
+    width: 97.8%;
+    margin-left: 1.8em;
+  }
+}
+
+.home-list-control, .dashboard-list-control {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+
+  h1.collapsible-header {
+    border-bottom: none;
+  }
+
+  h1.collapsible-header a:not(.dashlet-title), h1.dashlet-header a:not(.dashlet-title) {
+    margin-left: .3em;
+    font-weight: normal;
+    color: @icinga-blue;
+  }
+
+  h1.dashlet-header a.dashlet-title {
+    color: @text-color;
+  }
+
+  .dashlets-list-info, .dashboard-list-info {
+    .expand-icon {
+      margin-top: .9em;
+    }
+
+    .collapse-icon {
+      margin-top: .6em;
+    }
+  }
+}
+
+// Dashboard home list controls only
+.home-list-control {
+  width: 100%;
+
+  & > h1.collapsible-header {
+    margin-left: 1.5em;
+    margin-bottom: -1em;
+  }
+
+  .dashboard-list-control, .action-link.add-dashboard {
+    margin-left: 1.4em;
+  }
+
+  .dashboard-list-control {
+    .dashboard-item-list, .dashlet-item-list {
+      margin: 0;
+    }
+
+    .dashlet-item-list {
+      margin-left: 2em;
+      width: 98%; // 100% - 2em margin-left
+    }
+
+    .action-link.add-dashlet {
+      margin-left: 1.6em;
+      width: calc((49.74% * 2) - 1.2em); // 2x flex columns - own size
+    }
+  }
+}
+
+// Add dashlets mini wizard
+.form-list-control {
+  span {
+    font-size: 1.3em;
+    opacity: .8;
+    font-weight: 500;
+  }
+}
+
+.form-list-control.multi-dashlets {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  border-top: 2px solid @gray-light;
+
+  .control-group {
+    width: 100%;
+  }
+}
+
+.form-list-control.multi-dashlets:not(.collapsed) span {
+  margin-left: 1.4em;
+}
+
+// Make the submit buttons a bit bigger than the normal size
+.modal-form.icinga-controls .control-group.form-controls input[type="submit"],
+.control-group.form-controls .modal-cancel {
```

Comment:
This is a reason to refactor the general form control styles in my opinion.

You have a button inside `.form-controls`. This makes your button a form control, as such it should follow the same layout rules as all the other form controls.
Since your `.modal-cancel` button is a `button[type=button]`, this doesn't happen. `button[type=button]` should also get the same layout rules if inside `.form-controls`. Adjust the file form.less for this and remove the this section here.

Reply:
The point here is not that my `.modal-cancel` doesn't have the same layout as all other from-controls, but rather the overall buttons size no matter what kind. All form-controls submit buttons don't look as big by default as in the mockups in the referenced issue.

Reply:
Ooof, mockups are an artist's interpretation. Just like artworks. Well, no, artworks are way more sophisticated.. Didn't you notice that some of the button labels are randomly placed? These are quickly assembled, to get a rough idea how something should look like. Most of the time this is about the general layout, colors and such. But of course, this is a part of a broader application, that has it's default styles. These modals are in no way expected to **not** use these default styles. It's the exact opposite, they **have to** to use them, where applicable.

If a mockup looks 100% like the end result. The mockup is good, not the end result.